dev_guide/cron_jobs.adoc

```diff
@@ -70,14 +70,79 @@ spec:
   jobTemplate:             <2>
     spec:
       template:
+        metadata:
+          labels:          <3>
+            parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: Never
+          restartPolicy: OnFailure <4>
 ----
 
-1. Schedule for the job. In this example, a job will run every minute.
+1. Schedule for the job. In this example, the job will run every minute.
 2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
+3. Sets a label for jobs spawned by this cron job. 
+4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
 ====
+
+Alternatively, you can create and launch a cron job from a single command -- specifically, `oc run`. The following command creates and launches the same cron job as the `pi` resource:
```

Comment:
Suggestion: Removing alternatively, specifically (Minimalism)
```
You can also create and launch a cron job from a single command using `oc run`. The following command creates and launches the same cron job as specified in the previous example:
```
the example says "a CronJob resource:" We can change the example to say something similar to "a CronJob resource called pi (or named pi)"   and then use `cron job as the `pi` resource`